coralogix/resource_coralogix_alert.go

```diff
@@ -1163,43 +1163,42 @@ func metricSchema() map[string]*schema.Schema {
 								"less_than": {
 									Type:     schema.TypeBool,
 									Optional: true,
-									ExactlyOneOf: []string{"metric.0.promql.0.condition.0.more_than",
-										"metric.0.promql.0.condition.0.less_than",
+									ExactlyOneOf: []string{
+										"metric.0.promql.0.condition.0.more_than",
 										"metric.0.promql.0.condition.0.more_than_usual",
-										"metric.0.promql.0.condition.0.less_than_usual"},
+										"metric.0.promql.0.condition.0.less_than_usual",
+										"metric.0.promql.0.condition.0.more_than_or_equal",
+										"metric.0.promql.0.condition.0.less_than_or_equal",
+									},
 									Description: "Determines the condition operator." +
 										" Must be one of - less_than or more_than.",
 								},
 								"more_than": {
 									Type:     schema.TypeBool,
 									Optional: true,
-									ExactlyOneOf: []string{"metric.0.promql.0.condition.0.more_than",
-										"metric.0.promql.0.condition.0.less_than",
-										"metric.0.promql.0.condition.0.more_than_usual",
-										"metric.0.promql.0.condition.0.less_than_usual"},
 									Description: "Determines the condition operator." +
```

Comment:
Shouldn't the description be updated? As far as I understand the `ExactlyOneOf` block is gone

Reply:
The other validations were unnecessary, as ExactlyOneOf already checks that exactly one of those parameters were defined, and it doesn't have to be check for every one of the same parameters.